### ISSUE TITLE
CentOS 7 install fails due to lldpd not being installed

### ISSUE THREAD

**Issue by alikhajeh1** [pyguy/ansible-hpe-software#3]
When deploying this ansible role on an HPE G9 server with CentOS 7, I got the following error:
```
TASK [hpe-software : Install HPE Softwares] ****************************************************************************************************************************************************************
fatal: [x]: FAILED! => {"changed": false, "msg": "No package matching 'lldpd' found available, installed or updated", "rc": 126, "results": ["No package matching 'lldpd' found available, installed or updated"]}
```

**Comment by pyguy** [comment]
I tested on CentOS Linux release 7.5.1804, the name of the package is `lldpd`, it's a little awkward, anyway if you need to install it without `lldpd` package use this tag: `v1.0.2`

**Comment by pyguy** [comment]
the issue was regarded to the `epel` repo that should be installed for `lldpd` package, I will fix this ASAP.

**Comment by alikhajeh1** [comment]
Thanks @pyguy!

**Comment by pyguy** [comment]
my pleasure! @alikhajeh1 

I fixed it on tag `v1.2.4`, you can use it now.
